src/v8n.test.js

```diff
@@ -890,6 +875,18 @@ describe("rules", () => {
       expect(validation.test("Hello")).toBeFalsy();
     });
   });
+
+  test("optional", () => {
+    const v = v8n().optional(
```

Comment:
This is not a change request but just a suggestion. We're used to using `validation` name in all tests, maybe you should consider changing `v` to `validation`. But this is a not a big deal.